src/server/mods.cpp

```diff
@@ -98,6 +98,7 @@ void ServerModManager::getModNames(std::vector<std::string> &modlist) const
 
 void ServerModManager::getModsMediaPaths(std::vector<std::string> &paths) const
 {
+	paths.push_back(getBuiltinLuaPath() + DIR_DELIM + "locale");
```

Comment:
Doing this here breaks the unittests: https://github.com/minetest/minetest/blob/07500479191ed927ab661b3758ffcd2fd43158c5/src/unittest/test_servermodmanager.cpp#L158-L164

But regardless of that it'd just be cleaner to do this in [`Server::fillMediaCache()`](https://github.com/minetest/minetest/blob/375bcd65c1903957e3a640cefffcc8df164b78bd/src/server.cpp#L2493)